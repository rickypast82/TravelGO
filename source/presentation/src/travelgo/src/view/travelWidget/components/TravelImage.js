//@flow
import styled from 'styled-components'

const TravelImage = styled.img`
    width: 100%;
    border-radius: 3px;
`;

export default TravelImage;