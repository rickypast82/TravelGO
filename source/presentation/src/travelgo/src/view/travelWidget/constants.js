//@flow
type SmallSize = 'SMALL@travelWidget';
export const SMALL_SIZE:SmallSize = 'SMALL@travelWidget';

type MediumSize = 'MEDIUM@travelWidget';
export const MEDIUM_SIZE:MediumSize = 'MEDIUM@travelWidget';

export type SizeTravelWidget = SmallSize|MediumSize;