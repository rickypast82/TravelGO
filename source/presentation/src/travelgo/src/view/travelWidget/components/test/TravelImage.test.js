//@flow
/*eslint no-magic-numbers: "off"*/
import {getWidth} from '../TravelImage'
import { SMALL_SIZE, MEDIUM_SIZE} from '../../constants'

describe('TravelImage components test', () => {
    
    it('getWidth from imageSize small', () => {
        const size = getWidth(SMALL_SIZE);
        expect(size).toEqual(15);
    });

    it('getWidth from imageSize medium', () => {
        const size = getWidth(MEDIUM_SIZE);
        expect(size).toEqual(25);
    });

    it('getWidth from imageSize null', () => {
        const size = getWidth(null);
        expect(size).toEqual(15);
    });

});
