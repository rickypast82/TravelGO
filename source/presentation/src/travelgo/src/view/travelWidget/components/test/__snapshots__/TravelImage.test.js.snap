// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TravelImage components test renders without crashing with all props 1`] = `
<Image
  as="img"
  rounded={true}
  src="Roma.jpg"
  ui={true}
/>
`;
